src/basic/renderers/boolean.ts

```diff
@@ -21,7 +21,8 @@ import { renderErrors } from './errors';
  * Renders a labeled checkbox and any policy failure messages.
  */
 class BooleanAttributeCallbackRenderer
-  implements DestroyableCallbackRenderer, FocusableCallbackRenderer {
+  implements DestroyableCallbackRenderer, FocusableCallbackRenderer
+{
```

Comment:
These three files are automatically reformatted because of Prettier. I'm not sure why it's now formatting it with the brace on a new line in just these three instances, but it wasn't straight forward to alter this behavior, so I just left it as is.